lib/transproc/hash.rb

```diff
@@ -333,6 +333,23 @@ def self.unwrap(hash, root, keys = nil, prefix: false)
       unwrap!(copy, root, keys, prefix: prefix)
     end
 
+    # Same as `:unwrap` but handles nil values
+    #
+    # If value for a specified key is nil, or there is no such key in a hash
+    # `:try_unwrap` returns hash without the specified key and with no added element
+    #
+    # @see HashTransformations.unwrap
+    #
+    # @api public
+    def self.try_unwrap(hash, root, keys = nil, prefix: false)
```

Comment:
btw, what's the application of mutating versions? `transproc` feeds output of method to the next method, so mutating isn't even needed